client/post-editor/restore-post-dialog.jsx

```diff
@@ -100,10 +101,10 @@ class EditorRestorePostDialog extends Component {
 export default connect( state => {
 	const siteId = getSelectedSiteId( state );
 	const postId = getEditorPostId( state );
-	const capabilities = getEditedPostValue( state, siteId, postId, 'capabilities' );
```

Comment:
The `getEditedPostValue` selector retrieves the `capabilities` of a particular post. You are replacing it with the site capabilities. Are you sure they are equivalent. Can I have the capability to delete one post and not delete another on the same site?

Reply:
Ah, nice catch. 👍